sandbox/src/main/java/com/badoo/ribs/sandbox/rib/switcher/SwitcherInteractor.kt

```diff
@@ -104,6 +114,7 @@ internal class SwitcherInteractor(
         }
     }
 
+    // TODO CALL SUPER, LINT HIGHLIGHTS IT, BUT CAN BE EASILY MISSED
```

Comment:
It is very important to call `super` here, otherwise it won't work.
IDEA highlights it, `lintDebug` reports error, and can be discovered via test.